packages/storage/lib/impl/exampleThemeStorage.ts

```diff
@@ -1,6 +1,12 @@
-import { createStorage } from './base';
-import { StorageEnum } from './enums';
-import type { Theme, ThemeStorage } from './types';
+import { StorageEnum } from '../base/enums';
+import { createStorage } from '../base/base';
+import type { BaseStorage } from '../base/types';
+
+type Theme = 'light' | 'dark';
+
+type ThemeStorage = BaseStorage<Theme> & {
+  toggle: () => Promise<void>;
+};
```

Comment:
Why are you moved it from `types.ts`?
That file was created for store all kind of types from `storage`

Reply:
Because It's related to user specific storage model and will be hassle to rebase or merge upstream to fork if smth changes on storage generics.

Reply:
ok, that make sense